CMakeLists.txt

```diff
@@ -0,0 +1,147 @@
+cmake_minimum_required(VERSION 2.8)
```

Comment:
This very much looks like a portable cmake file, if this works i will convert over some of the upstream stuff.

Reply:
It worked on windows for me, but I don't think I ever needed it on linux